command/secrets/output/result.go

```diff
@@ -0,0 +1,70 @@
+package output
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command/helper"
+)
+
+type SecretsOutputResult struct {
+	Address   string `json:"address,omitempty"`
+	BLSPubkey string `json:"bls,omitempty"`
+	NodeID    string `json:"node_id,omitempty"`
+
+	outputNodeID    bool `json:"-"`
+	outputBLS       bool `json:"-"`
+	outputValidator bool `json:"-"`
+}
+
+func (r *SecretsOutputResult) GetOutput() string {
+	var buffer bytes.Buffer
+
+	if r.outputNodeID {
+		buffer.WriteString(r.NodeID)
+
+		return buffer.String()
```

Comment:
Why not just return the value? Buffer is used to concat string.

Reply:
Resolved in https://github.com/0xPolygon/polygon-edge/pull/705/commits/70f9884642c307c03187adb82b0e7c24cb85051f